docs/SOAP-Service-Methods.md

```diff
@@ -1063,7 +1063,7 @@ Get the status change log for given activation and date range.
 | `Long` | `id` | Change ID |
 | `String` | `activationId` | An identifier of an activation |
 | `ActivationStatus` | `activationStatus` | An activation status at the moment of a signature verification |
-| `String` | `blockedReason` | Reason why activation was blocked (default: NOT_SPECIFIED) |
+| `String` | `eventReason` | Reason why this activation history record was created (default: null or NOT_SPECIFIED) |
```

Comment:
It is a bit weird to provide 2 default values.

Reply:
That's how it is in the database.

Reply:
OK, I'll change that to `null`. It has now more wider meaning than before and for the most of events it's `null`